cmd/attack/jvm.go

```diff
@@ -0,0 +1,183 @@
+// Copyright 2021 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package attack
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"go.uber.org/fx"
+
+	"github.com/chaos-mesh/chaosd/cmd/server"
+	"github.com/chaos-mesh/chaosd/pkg/core"
+	"github.com/chaos-mesh/chaosd/pkg/server/chaosd"
+	"github.com/chaos-mesh/chaosd/pkg/utils"
+)
+
+func NewJVMAttackCommand() *cobra.Command {
+	options := core.NewJVMCommand()
+	dep := fx.Options(
+		server.Module,
+		fx.Provide(func() *core.JVMCommand {
+			return options
+		}),
+	)
+
+	cmd := &cobra.Command{
+		Use:   "jvm <subcommand>",
+		Short: "JVM attack related commands",
+	}
+
+	cmd.AddCommand(
+		NewJVMInstallCommand(dep, options),
+		NewJVMSubmitCommand(dep, options),
+	)
+
+	return cmd
+}
+
+func NewJVMInstallCommand(dep fx.Option, options *core.JVMCommand) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "install [options]",
+		Short: "install agent to Java process",
+		Run: func(*cobra.Command, []string) {
+			options.Type = core.JVMInstallType
+			options.CompleteDefaults()
```

Comment:
It seems that core.JVMCommand not implement this func. Is it really necessary here ?

Reply:
I think it's better to remain it, and I implement this function in 3a80f1e

Reply:
OK